pkg/azuredisk/nodeserver.go

```diff
@@ -444,16 +445,16 @@ func (d *Driver) NodeGetInfo(ctx context.Context, _ *csi.NodeGetInfoRequest) (*c
 	}, nil
 }
 
-func getMaxDataDiskCount(instanceType string) int64 {
+func getMaxDataDiskCount(instanceType string) (int64, bool) {
```

Comment:
can you add ut for returned bool value in another PR? thx

Reply:
Added : https://github.com/kubernetes-sigs/azuredisk-csi-driver/pull/3080